.circleci/config.yml

```diff
@@ -0,0 +1,31 @@
+version: 2.1
+
+orbs:
+  elixir-orb: fresha/elixir-orb@0.0
+
+workflows:
+  version: 2
+  ci:
+    jobs:
+      - elixir-orb/test:
+          executor: elixir-orb/elixir-postgres
```

Comment:
Is postgres required?
If not you could use executor without it
https://github.com/surgeventures/elixir-orb/tree/master/src/executors